src/managers/UserManager.js

```diff
@@ -48,6 +51,7 @@ class UserManager extends BaseManager {
   resolveID(user) {
     if (user instanceof GuildMember) return user.user.id;
     if (user instanceof Message) return user.author.id;
+    else if (typeof user === 'string') return user.match(USERS_PATTERN)[1];
```

Comment:
This will throw `Cannot read property '1' of null` if the regex doesn't match

Reply:
same as above